### DISCUSSION TITLE
Log P

### DISCUSSION THREAD

**Discussion by Demydoff** [abrechts/Phoenix-ELN#23] [Feature Proposals]
![image](https://github.com/user-attachments/assets/5f6909d5-98bb-4f8f-aa38-bd643949ff16)
You could add the value of log P in this column. This would allow you to choose the method of isolating the product based on this value, or you would not need to search the values ​​on the internet or open another chemistry software.

**Comment by abrechts** [comment]
Unfortunately, the technology behind ChemBytes Draw does not allow the calculation of log P values. However, for known substances, you still may be able to obtain these from the PubChem and/or ChemSpider links.

**Reply by Demydoff** [reply]
Thank you for the reply!
